test/widgets/inbox_test.dart

```diff
@@ -310,6 +345,30 @@ void main() {
           checkAppearsUncollapsed(tester, findSectionContent);
         });
 
+        testWidgets('collapse all-DMs section after scroll', (tester) async {
+          await setupVarious(tester);
+
+          final listFinder = find.byType(Scrollable);
+          final findSectionContent =
+              find.text(eg.otherUser.fullName).hitTestable();
```

Comment:
nit: two-space indents:
```suggestion
          final findSectionContent =
            find.text(eg.otherUser.fullName).hitTestable();
```

(Better yet, give a shorter name like `findContent` and then this fits on one line. But in any case there are some other spots that should be two-space indents too.)

Reply:
Sorry for this inconvenience each time. As the project doesn't use the `dart auto-formatter`, sometimes it stays out of sight.